diff --git a/node_modules/coco-cashu-core/dist/index.d.cts b/node_modules/coco-cashu-core/dist/index.d.cts
index 39ff868..4b9d00b 100644
--- a/node_modules/coco-cashu-core/dist/index.d.cts
+++ b/node_modules/coco-cashu-core/dist/index.d.cts
@@ -1425,7 +1425,7 @@ declare class PaymentRequestService {
 declare class WalletApi {
   private mintService;
   private walletService;
-  private proofService;
+  proofService: ProofService;
   private walletRestoreService;
   private transactionService;
   private paymentRequestService;
@@ -1797,7 +1797,7 @@ declare class Manager {
   readonly send: SendApi;
   private mintService;
   private walletService;
-  private proofService;
+  proofService: ProofService;
   private walletRestoreService;
   private keyRingService;
   private eventBus;
@@ -1811,7 +1811,7 @@ declare class Manager {
   private meltQuoteService;
   private historyService;
   private seedService;
-  private counterService;
+  counterService: CounterService;
   private transactionService;
   private paymentRequestService;
   private sendOperationService;
diff --git a/node_modules/coco-cashu-core/dist/index.d.ts b/node_modules/coco-cashu-core/dist/index.d.ts
index 39ff868..4b9d00b 100644
--- a/node_modules/coco-cashu-core/dist/index.d.ts
+++ b/node_modules/coco-cashu-core/dist/index.d.ts
@@ -1425,7 +1425,7 @@ declare class PaymentRequestService {
 declare class WalletApi {
   private mintService;
   private walletService;
-  private proofService;
+  proofService: ProofService;
   private walletRestoreService;
   private transactionService;
   private paymentRequestService;
@@ -1797,7 +1797,7 @@ declare class Manager {
   readonly send: SendApi;
   private mintService;
   private walletService;
-  private proofService;
+  proofService: ProofService;
   private walletRestoreService;
   private keyRingService;
   private eventBus;
@@ -1811,7 +1811,7 @@ declare class Manager {
   private meltQuoteService;
   private historyService;
   private seedService;
-  private counterService;
+  counterService: CounterService;
   private transactionService;
   private paymentRequestService;
   private sendOperationService;
