diff --git a/node_modules/heroui-native/lib/module/helpers/internal/components/full-window-overlay.js b/node_modules/heroui-native/lib/module/helpers/internal/components/full-window-overlay.js
index e02d2f8..15ee3d9 100644
--- a/node_modules/heroui-native/lib/module/helpers/internal/components/full-window-overlay.js
+++ b/node_modules/heroui-native/lib/module/helpers/internal/components/full-window-overlay.js
@@ -35,6 +35,7 @@ export function FullWindowOverlay({
     });
   }
   return /*#__PURE__*/_jsx(NativeFullWindowOverlay, {
+    unstable_accessibilityContainerViewIsModal: false,
     children: children
   });
 }
diff --git a/node_modules/heroui-native/src/helpers/internal/components/full-window-overlay.tsx b/node_modules/heroui-native/src/helpers/internal/components/full-window-overlay.tsx
index 50ac630..0f55a58 100644
--- a/node_modules/heroui-native/src/helpers/internal/components/full-window-overlay.tsx
+++ b/node_modules/heroui-native/src/helpers/internal/components/full-window-overlay.tsx
@@ -44,5 +44,5 @@ export function FullWindowOverlay({
     return <>{children}</>;
   }
 
-  return <NativeFullWindowOverlay>{children}</NativeFullWindowOverlay>;
+  return <NativeFullWindowOverlay unstable_accessibilityContainerViewIsModal={false}>{children}</NativeFullWindowOverlay>;
 }
